docs/package.json

```diff
@@ -25,7 +25,7 @@
     "@emotion/server": "11.11.0",
     "@emotion/styled": "11.11.5",
     "@mui/base": "5.0.0-beta.40",
-    "@mui/docs": "6.0.0-dev.240424162023-9968b4889d",
+    "@mui/docs": "6.0.0-alpha.13",
```

Comment:
`import { SectionTitle } from '@mui/docs/SectionTitle';` probably worked only because of the present alias.
Bumped the dependency version where this component is exported.